query/stdlib/influxdata/influxdb/source.go

```diff
@@ -84,6 +74,30 @@ func (s *Source) processTables(ctx context.Context, tables TableIterator, waterm
 	return nil
 }
 
+func (s *Source) processTable(ctx context.Context, tbl flux.Table) error {
+	if len(s.ts) == 0 {
+		tbl.Done()
+		return nil
+	} else if len(s.ts) == 1 {
+		return s.ts[0].Process(s.id, tbl)
```

Comment:
I see. So if there are multiple downstream transformations, for each table we send copies downstream and then release our specific instance. But if there's only one downstream transformation, there's no need to copy the table, we just send it downstream and it is the job of that transformation to release it. Correct?

Reply:
Yes. That is correct.